kie-server-parent/kie-server-controller/kie-server-controller-test-war/src/main/assembly/assembly-ee6-container.xml

```diff
@@ -50,6 +50,7 @@
 
         <include>com.sun.xml.bind:jaxb-impl</include>
         <include>com.sun.xml.bind:jaxb-core</include>
+        <include>org.slf4j:slf4j-jdk14</include>
```

Comment:
Without this dependency WebSphere fails to deploy RestKieServerControllerImpl with:
 Caused by: java.lang.ClassNotFoundException: org.slf4j.Logger